lib/validators.js

```diff
@@ -97,11 +96,11 @@ const create = function (options = {}) {
                 request[p][parameter.name] = coerce && request[p][parameter.name] && coerce(request[p][parameter.name]);
                 return h.continue;
             },
-            validate: async function (value) {
```

Comment:
i imagine this was a left over from earlier version of async joi validate calls.